Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN mkdir .setup
 ADD environment-python2.yml .setup/
 RUN conda env create -n python2 -f .setup/environment-python2.yml && \
     # Jupyterlab component for ipywidgets (must match jupyterlab version) \
-    jupyter labextension install @jupyter-widgets/jupyterlab-manager@^0.36
+    jupyter labextension install @jupyter-widgets/jupyterlab-manager
```

Comment:
It's important for the jupyter-widgets/jupyterlab-manager version to _exactly_ match the jupyterhub version:
https://github.com/jupyter-widgets/ipywidgets/tree/master/packages/jupyterlab-manager#version-compatibility

The jupyterlab version is determined by the base image [`jupyter/base-notebook`](https://hub.docker.com/r/jupyter/base-notebook/tags/). Since we're updating dependencies this would be a good time to update the base image too.

Reply:
I removed that one by mistake but I agree that we can look at upgrading the base image